src/index.css

```diff
@@ -0,0 +1,12 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
```

Comment:
- [ Required ] I think it would be nice if the `font` in use in your App is consistent with the font in the Behance design which is `Lato` or `Gill sans`, currently, your App does not use any of these fonts. Please see the screenshot below on how the fonts should be used and rectify this.

## Fonts
![Screenshot from 2023-09-06 06-29-24](https://github.com/noelfoka/react-capstone-project/assets/16392046/467f5ed4-1eda-491d-8a38-cb6e61887771)